translations/default/AdminCatalogFeature.xlf

```diff
@@ -2111,65 +2343,65 @@
       <trans-unit id="9b9237161ab556e135d50c3384ba6599">
         <source>Related product</source>
         <target>Related product</target>
-        <note>Line: 27</note>
+        <note>Line: 26</note>
       </trans-unit>
       <trans-unit id="0e7abe02d1bb48b126ffaae21749ea13">
         <source>Add a related product</source>
         <target>Add a related product</target>
-        <note>Line: 45</note>
+        <note>Line: 43</note>
       </trans-unit>
     </body>
   </file>
   <file original="src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_seo.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="f5a592018c8b55d2c968515ad7c58c91">
-        <source>Search Engine Optimization</source>
-        <target>Search Engine Optimization</target>
-        <note>Line: 27</note>
+      <trans-unit id="c6b3d0f163e32b16cbdde201859d33e3">
+        <source>Here is a preview of your search engine result, play with it!</source>
+        <target>Here is a preview of your search engine result, play with it!</target>
+        <note>Line: 30</note>
       </trans-unit>
       <trans-unit id="926b7b9f596e468ebce7da2c42c1e27c">
         <source>Improve your ranking and how your product page will appear in search engines results.</source>
         <target>Improve your ranking and how your product page will appear in search engines results.</target>
-        <note>Line: 28</note>
-      </trans-unit>
-      <trans-unit id="e1be175ec06a5890119e29ffdb34631e">
-        <source>Reset URL</source>
-        <target>Reset URL</target>
-        <note>Line: 67</note>
+        <note>Line: 27</note>
       </trans-unit>
       <trans-unit id="e642dbf4ded4c06ae9feae332a0a83c4">
         <source>Redirection page</source>
         <target>Redirection page</target>
-        <note>Line: 96</note>
+        <note>Line: 97</note>
       </trans-unit>
-      <trans-unit id="c6b3d0f163e32b16cbdde201859d33e3">
-        <source>Here is a preview of your search engine result, play with it!</source>
-        <target>Here is a preview of your search engine result, play with it!</target>
-        <note>Line: 31</note>
+      <trans-unit id="e1be175ec06a5890119e29ffdb34631e">
+        <source>Reset URL</source>
+        <target>Reset URL</target>
+        <note>Line: 68</note>
+      </trans-unit>
+      <trans-unit id="f5a592018c8b55d2c968515ad7c58c91">
+        <source>Search Engine Optimization</source>
+        <target>Search Engine Optimization</target>
+        <note>Line: 26</note>
       </trans-unit>
     </body>
   </file>
   <file original="src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_shipping.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
+      <trans-unit id="fb6d68835802519a7f425ca216778b15">
+        <source>Does this product incur additional shipping costs?</source>
```

Comment:
It should look like this and so it should be "Additional shipping costs"
<img width="571" alt="Capture d’écran 2022-07-28 à 14 36 56" src="https://user-images.githubusercontent.com/70583503/181506578-47b9ac74-75c2-4092-a9af-66128f535d50.png">